srcpkgs/tigervnc/template

```diff
@@ -58,6 +72,12 @@ post_build() {
 }
 
 post_install() {
+	# More Void customizations
+	cd ${FILESDIR}/../..
+	vinstall lightdm/files/xsession 755 etc/tigervnc Xsession-fallback
+	# runit unit for display 1
+	vsv vncserver-1
```

Comment:
I think I can make it work that way. It may take a little time for me to test. I thought the way agetty-ttyN services do it was OK. I think I will just have the vncserver service always be for display :1, but it seems the services on the other displays will need to have a different name such as vncserver-2, vncserver-3, and I can explain how to create them in README.voidlinux. I might spend a little time thinking if it is possible for a single vncserver service to run more than one display and implement that if possible. I will have to think about it.

Reply:
This is working now with a vncserver service instead of a vncserver-1 service. I also updated README.voidlinux to explain how to add additional servers without using symbolic links.